x-pack/test/typings/supertest_as_promised.d.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+declare module 'supertest-as-promised';
```

Comment:
Super-curious whether this is needed, or it helps in some fashion with the TS typings, because it doesn't immediately look like it adds a ton of value :-).  Guessing it's just some linking to a separating typing file for the package, needed by our current ts config setup.

Reply:
Sadly, this seems to be a TS requirement - it won't let me change the *.js file to *.ts unless every one of it's imports has at lease a module declaration.

It obviously doesn't provide any definitions, so any use of code from the module isn't type checked, but it seems to be a requirement.